dev/io.openliberty.persistence.3.2.thirdparty/resources/org/eclipse/persistence/exceptions/i18n/ValidationExceptionResource.nlsprops

```diff
@@ -633,9 +633,12 @@
 
 7356=Procedure: [{1}] cannot be executed because {0} does not currently support multiple out parameters
 
-7357=The [{0}] URL for the [{1}] resource does not belong to a valid persistence root, as defined by the JPA Specification.
+7357=The "[{0}]" URL for the "[{1}]" resource does not belong to a valid persistence root, as defined by the Jakarta Persistence specification
 
 7358=Incorrect ASM service name provided.
 
 7359=No any ASM service available.
 
+7360=Database password was encrypted by deprecated algorithm.
```

Comment:
Suggested change:
The database password was encrypted by a deprecated algorithm.
It's recommended to reencrypt it by `passwordUpdate.sh` from an eclipselink.zip bundle

(We try not to use "recommended". The wording would be better if it were "Reencrypt it by ...". Add a period to the second sentence if it's needed.)